ctk/features/try.feature

```diff
@@ -15,21 +15,24 @@ Feature: Try Task
       namespace: default
       name: try-catch-404
     do:
-      try:
-        call: http
-        with:
-          method: get
-          endpoint:
-            uri: https://petstore.swagger.io/v2/pet/getPetByName/{petName}
-      catch:
-        errors:
-          with:
-            type: https://serverlessworkflow.io/dsl/errors/types/communication
-            status: 404
-        as: err
-        do:
-          set:
-            error: ${ $err }
+      - tryGetPet:
```

Comment:
After watching the examples updated I think it might be worth to reevaluate the need for enforcing names in task, see https://github.com/serverlessworkflow/specification/issues/849 and https://github.com/serverlessworkflow/specification/discussions/886
Compare 
```
do:
   -tryGetPet:
       try: 
           -getPet:
               call: http
                with:
                  method: get
                  endpoint:
                    uri: https://petstore.swagger.io/v2/pet/getPetByName/{petName}
```
with 
```
do:   
   - try:
      name: getPet
      call: http
      with:
         method: get
                  endpoint:
                    uri: https://petstore.swagger.io/v2/pet/getPetByName/{petName} 
```

Reply:
I understand your point of view. However,  it remains the same than discussed before, I'd rather keep names as non-optional.

Reply:
Please, leave discussions in the issues. PR are about points agreed upon beforehand.

Reply:
Sure, I did not want to reopen a closed discussion just for the sake of discussing endlessly ;). 
I just realize that when using try with just one task, it is difficult to give different names to the try task and the task within the try. I feel most users will end up typing "<taskName>"  for the task and "try<TaskName" for the try. 
When having multiple task in the try, it becomes event more difficult to find a name for the try task, because user has "<taskName1>" and "<taskName2>" (thats ok and mandatory to distinguish the tasks), but to name the try user will have to summarize the two tasks into one meaningful name. 
Not a big deal anyway. Lets move on.